scapy/contrib/loraphy2wan.py

```diff
@@ -623,10 +633,10 @@ class RejoinReq(Packet):  # LoRa 1.1 specs
 
 class FRMPayload(Packet):
     name = "FRMPayload"
-    fields_desc = [ConditionalField(StrField("DataPayload", "", remain=4),  # Downlink  # noqa: E501
```

Comment:
A `MultipleTypeField` would probably fit better instead of renaming those fields.

Reply:
I wasn't aware of that one! Thanks, I'll use it instead :)